web/packages/teleport/src/Login/Login.tsx

```diff
@@ -78,3 +90,114 @@ export function LoginComponent({
     </>
   );
 }
+
+const LicenseBox = styled(Box)`
+  width: 100%;
+  max-width: 550px;
+  padding: ${props => props.theme.space[4]}px;
+  background-color: ${props => props.theme.colors.levels.surface};
+  border-radius: ${props => props.theme.radii[3]}px;
+  margin: auto;
+`;
+
+function LicenseAcknowledgement({
+  setLicenseAcknowledged,
+}: {
+  setLicenseAcknowledged(): void;
+}) {
+  const [checked, setChecked] = useState(false);
+
+  return (
+    <>
+      <LogoHero />
+      <LicenseBox>
+        <Text
+          css={`
+            line-height: 32px;
+            font-weight: 500;
+            font-size: ${props => props.theme.fontSizes[7]}px;
+          `}
+        >
+          Welcome to Teleport
+        </Text>
+        <Text
+          css={`
+            font-weight: 300;
+            line-height: 24px;
+            font-size: ${props => props.theme.fontSizes[3]}px;
+            color: ${p => p.theme.colors.text.slightlyMuted};
+          `}
+        >
+          Companies may use Teleport Community Edition on the condition they
+          have less than 100 employees and less than $10MM in annual revenue. If
+          your company exceeds these limits, please{' '}
+          <Text
+            as="a"
+            href="https://goteleport.com/signup/enterprise/?utm_campaign=CTA_terms_and_conditions&utm_source=oss&utm_medium=in-product"
+          >
+            contact us
+          </Text>{' '}
+          to evaluate and use Teleport.
+        </Text>
+        <Flex mt={3} alignItems="center" gap={2}>
+          <StyledCheckbox
+            checked={checked}
+            onChange={e => {
+              setChecked(e.target.checked);
+            }}
+          />
+          By clicking continue, you agree to our{' '}
+          <Text
+            as="a"
+            href="https://github.com/gravitational/teleport/blob/master/LICENSE-community"
+          >
+            Terms and Conditions
+          </Text>
+        </Flex>
+        <ButtonPrimary
+          disabled={!checked}
+          onClick={() => {
+            setLicenseAcknowledged();
+            window.location.reload();
+          }}
+          textTransform="none"
+          block
+          mt={3}
+        >
+          Continue
+        </ButtonPrimary>
+      </LicenseBox>
+      <Flex
+        css={`
+          width: 100%;
+          position: absolute;
+          bottom: 24px;
+          gap: 45px;
+          justify-content: center;
+        `}
+      >
+        <Text>@Gravitational, Inc. All Rights Reserved</Text>
```

Comment:
This this be a `©` instead of an `@`?

Reply:
Thank you. I just copy pasted and I'm sure it was a placeholder